sdks/python/apache_beam/internal/json_value.py

```diff
@@ -20,6 +20,10 @@
 from apitools.base.py import extra_types
 
 
+_MAXINT64 = 2 << 63 - 1
+_MININT64 = -(2 << 63)
```

Comment:
Actually, these values from apitools are wrong.  It should be "1 << 63" because of the sign bit.